src/os/exec_linux_test.go

```diff
@@ -0,0 +1,32 @@
+//go: build linux && !baremetal && !tinygo.wasm
+
+package os_test
+
+import (
+	"errors"
+	. "os"
+	"runtime"
+	"testing"
+)
+
+// Test the functionality of the forkExec function, which is used to fork and exec a new process.
+// This test is not run on Windows, as forkExec is not supported on Windows.
+// This test is not run on Plan 9, as forkExec is not supported on Plan 9.
+func TestForkExec(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		t.Logf("skipping test on %s", runtime.GOOS)
+		return
+	}
+
+	proc, err := StartProcess("/bin/echo", []string{"hello", "world"}, &ProcAttr{})
```

Comment:
Can you also add a negative test, to make sure that running `/bin/non-existing-program` for example returns the expected error message?

Can you also add some tests that verify that unsupported features like `ProcAttr.Dir`, `ProcAttr.Sys`, etc will return an error as they should? (These checks probably also need to be implemented). If a program sets `Dir` for example but then the program executes in another directory without error, that's going to be weird and difficult to debug. I'd rather not have users deal with such bugs.